src/mercator/mercator/static/js/Packages/Mercator/2016/Proposal/Module.ts

```diff
@@ -65,6 +67,10 @@ export var register = (angular) => {
             adhEmbedProvider.registerDirective("mercator-2016-proposal-detail");
             adhEmbedProvider.registerDirective("mercator-2016-proposal-listitem");
         }])
+        // FIXME: doesn't work
```

Comment:
The issue seems to be that `contentType` is not defined directly, so `getName()` is called with an empty first argument.